### ISSUE TITLE
Setting static IP for google_notebooks_instance

### ISSUE THREAD

**Issue by Apachaika** [terraform-google-modules/terraform-docs-samples#453]
### TL;DR

How to set static IP for vertex_ai_notebooks_instance?

### Terraform Resources

https://github.com/terraform-google-modules/terraform-docs-samples/tree/main/vertex_ai/user_managed_notebooks_instance

### Detailed design

I checked all documentation about how to create notebooks_instance via Terraform.
But I came across the fact that Vertex Ai Notebooks have Dynamic IPs that complicate the process of distributing access over IP. As a workaround, you can manually create a static IP and bind it in the Notebook VM itself.
But I need to be able to do this through Terraform. 
Since the implementation works in a manual format so it's possible via terraform format.
Maybe you have a work sample to set static IP?

**Comment by msampathkumar** [comment]
Hi @Apachaika Thank you for your request, let me check on this.

**Comment by glasnt** [comment]
There doesn't appear to be a way in https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/notebooks_instance to set this value. 

Since this is a value on the Compute Engine instance, you'd have to configure it explicitly in your Terraform to be able to have terraform control the value. You may be able to use https://github.com/GoogleCloudPlatform/terraformer to help you generate the config required, and `terraform import` to import the information of an existing instance, and then configure the settings that way. 

This is outside the scope of the docs samples to implement at this time.
